src/database.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+from typing import Any
+
+from sqlalchemy import (
+    Boolean,
+    Column,
+    CursorResult,
+    DateTime,
+    ForeignKey,
+    Identity,
+    Insert,
+    Integer,
+    LargeBinary,
+    MetaData,
+    Select,
+    String,
+    Table,
+    Update,
+    func,
+)
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from sqlalchemy.orm import sessionmaker
+
+from .config import settings
+
+# from constants import DB_NAMING_CONVENTION
+
+DATABASE_URL = str(settings.DATABASE_URL)
+
+engine = create_async_engine(DATABASE_URL)
+
+metadata = MetaData()
+
+event = Table(
+    "event",
+    metadata,
+    Column("id", Integer, Identity(), primary_key=True),
+    Column("product", String, nullable=False),
+    Column("pennkey", String, nullable=False),
```

Comment:
I think `nullable` can be true for `pennkey` (we don't necessarily need a user next to every statistic), such as for things we want to aggregate